js-inputs-and-strings/js-inputs-and-strings_multiple-inputs/js/index.js

```diff
@@ -14,3 +14,20 @@ Implement the following functionality:
 const firstInput = document.querySelector("[data-js=first-input]");
 const secondInput = document.querySelector("[data-js=second-input]");
 const button = document.querySelector("[data-js=button]");
+
+button.addEventListener("click", principal);
+
+function principal() {
+  secondInput.value = firstInput.value;
+  button.addEventListener("click", secondClick);
+}
+
+function secondClick() {
+  secondInput.value = secondInput.value.toUpperCase();
+  button.addEventListener("click", thirdClick);
+}
+
+function thirdClick() {
+  firstInput.value = secondInput.value;
+  secondInput.value = firstInput.value.toLowerCase();
```

Comment:
does it change the other way around then without defining values of the input text before?

Reply:
no, because this works just in the third click.